src/Application.php

```diff
@@ -160,7 +166,26 @@ protected function reorderColumns(CsvFile $csv, array $items): array
 
     protected function getInputCsv(string $tableId): CsvFile
     {
-        return new CsvFile($this['parameters']['data_dir'] . "/in/tables/" . $tableId . ".csv");
+        $filteredStorageInputMapping = array_filter($this['inputMapping']['tables'], function ($v) use ($tableId) {
```

Comment:
Data to zapisat na jeden riadok, pri `fn` netreba ani pouzit `use`:
```suggestion
        $filteredStorageInputMapping = array_filter($this['inputMapping']['tables'], fn($v) => $v['source'] === $tableId);
```

Alebo aspon:
```suggestion
        $filteredStorageInputMapping = array_filter($this['inputMapping']['tables'], function ($v) use ($tableId) {
            return $v['source'] === $tableId;
        });
```